.github/workflows/ci.yaml

```diff
@@ -102,6 +102,32 @@ jobs:
            && !contains(github.event.pull_request.body, '[skip test_windows]')
```

Comment:
```suggestion
           && !contains(github.event.pull_request.body, '[skip test_windows_fast]')
```

Reply:
I've removed the line, as now the fast Windows CI finishes in ~8 minutes, usually, there is little need to skip that.